mmdet/models/backbones/res2net.py

```diff
@@ -98,7 +98,7 @@ def __init__(self,
         self.stage_type = stage_type
         self.scales = scales
         self.width = width
-        delattr(self, 'conv2')
+        # delattr(self, 'conv2')
```

Comment:
May I ask why change this?

Reply:
This is the simplest solution for the bug #3714 .

Reply:
I have uncommented this line after syncing with the latest updates of MMDetection.